receiver/vcenterreceiver/README.md

```diff
@@ -0,0 +1,57 @@
+# vCenter Receiver
+
+This receiver fetches metrics from a vCenter or ESXi host running VMware vSphere APIs.
+
+Supported pipeline types: `metrics` and `logs`(not implemented quite yet)
+
+> :construction: This receiver is in **BETA**. Configuration fields and metric data model are subject to change.
+
+## Prerequisites
+
+This receiver has been built to support ESXi and vCenter versions:
+
+- 7.0
+- 6.7
+- 7.5
+
+A “Read Only” user assigned to a vSphere with permissions to the vCenter server, cluster and all subsequent resources being monitored. Must be specified in order for the receiver to retrieve information about them.
+
+## Configuration
+
+Configuration is split up into `metrics` and `logs`
+
+```yaml
+receivers:
+    vcenter:
+      metrics:
+      logs:
+```
+
+### Metrics Configuration
+
+| Parameter | Default | Type | Notes |
+| --- | --- | --- | --- |
+| endpoint |  | String | Endpoint to the vCenter Server or ESXi host that has the sdk path enabled. Required. The expected format is `<protocol>://<hostname>` <br><br> i.e: `https://vcsa.hostname.localnet` |
+| username |  | String | Required |
+| password |  | String | Required |
+| tls | | TLSClientSetting | Not Required. Will use defaults for [configtls.TLSClientSetting](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md). By default insecure settings are rejected and certificate verification is on. |
+| collection_interval | 2m | Duration | This receiver collects metrics on an interval. If the vCenter is fairly large, this value may need to be increased. Valid time units are `ns`, `us` (or `µs`), `ms`, `s`, `m`, `h` |
+
+### Example Configuration
+
+```yaml
+receivers:
+  vcenter:
+    metrics:
+      endpoint: http://localhost:15672
+      username: otelu
+      password: $VCENTER_PASSWORD
+      collection_interval: 5m
+      metrics: []
```

Comment:
I think I mentioned in the tracking issue that this should probably be called `settings`, since it's already inside of a struct called `metrics`.

Reply:
But also it doesn't need to be in the example since it's empty anyways.